.travis.yml

```diff
@@ -1,7 +1,15 @@
 dist: xenial
 language: python
 cache: pip
-matrix:
+addons:
+  homebrew:
+    update: true
+    packages:
+    - npm
+    - openssl
+    - pyenv
+    - pyenv-virtualenv
```

Comment:
Travis is already creating you a virtualenv...  Layering multiple venvs is not advised.

Reply:
it should if you have the language set to `python` but that's not available in `osx` no?

Reply:
You are in your own container.  You do not need a venv.

Reply:
Also pyenv will do the openssl thing for you.

Reply:
I wanted to speed up the openssl, didn't help I realize now :)